pkgs/development/libraries/boost/1.71.nix

```
@@ -0,0 +1,12 @@
+{ stdenv, callPackage, fetchurl, fetchpatch, ... } @ args:
+
+callPackage ./generic.nix (args // rec {
+  version = "1.71.0";
+
+  src = fetchurl {
+    #url = "mirror://sourceforge/boost/boost_1_71_0.tar.bz2";
+    url = "https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.bz2";
```

maybe just use the `urls` attribute?
```suggestion
    urls = [ 
      "mirror://sourceforge/boost/boost_1_71_0.tar.bz2"
      "https://dl.bintray.com/boostorg/release/1.71.0/source/boost_1_71_0.tar.bz2"
    ];
```